lib/versioner/staggered_test.go

```diff
@@ -75,3 +76,23 @@ func parseTime(in string) time.Time {
 	}
 	return t
 }
+
+func TestCreateVersionPath(t *testing.T) {
+	file := "myfile"
+
+	tmpDir, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	folderFs := fs.NewFilesystem(fs.FilesystemTypeBasic, tmpDir)
+
+	versioner := newStaggered(folderFs, map[string]string{
+		"fsType": "",
```

Comment:
You don't need to add the empty string to the map. If `"fsType"` is missing, that's the same as an empty string.